src/controller/http-api-router.js

```diff
@@ -27,6 +27,15 @@ class HttpApiRouter {
             },
             { rateLimit: true, requestSchema: this.jsonSchemaService.publishSchema() },
         );
+
+        this.httpClientModuleManager.post(
+            '/query',
+            (req, res) => {
+                this.searchController.handleHttpApiQueryRequest(req, res);
+            },
+            { rateLimit: true, requestSchema: this.jsonSchemaService.querySchema() },
```

Comment:
Are we sure we want to rate limit this operations ?

Reply:
No we don't i will remove rate limit